### DISCUSSION TITLE
CoinGecko API

### DISCUSSION THREAD

**Discussion by epparshall** [OpenBB-finance/OpenBB#1067] [Q&A]
I have been trying to view the cgtrending coins; however, I keep getting this error:

"Connection error. Couldn't connect to CoinGecko and scrape the data. Please visit CoinGecko site, and check if it's not under maintenance"

Everything else is working in the terminal besides this feature. Is there something wrong with the CoinGecko API, or do I need to fix something on my end?

**Comment by jmaslek** [comment]
This unfortunately seems to happen with them from time to time

**Comment by DidierRLopes** [comment]
Hi @epparshall ,

Has your experience on this regard improved?
